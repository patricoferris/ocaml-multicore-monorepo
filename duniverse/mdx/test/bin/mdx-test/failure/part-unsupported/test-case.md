Part selection for non-OCaml files is not supported.

``` file=casual-file.txt,part=part1
```
