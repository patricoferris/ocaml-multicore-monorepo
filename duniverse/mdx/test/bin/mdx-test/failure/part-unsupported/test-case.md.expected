[mdx] Fatal error: File "test-case.md", lines 3-5: invalid code block: `part` label is not allowed for non-OCaml include blocks.
